tests/yam/agama/validate_product_installed_agama.pm

```diff
@@ -0,0 +1,28 @@
+# SUSE's openQA tests
```

Comment:
it is not specific from agama, we could use this validation somewhere else.
I would suggest `tests/yam/validations/validate_product.pm`
and in the future we could unify with `tests/console/validate_product_installed_SLES.pm`